src/environments/environment.ts

```diff
@@ -10,7 +10,8 @@ export const environment = {
   production: false,
   version: env.npm_package_version + '-dev',
   fineractPlatformTenantId: 'default',  // For connecting to server running elsewhere update the tenant identifier
-  baseApiUrl: 'https://dev.mifos.io',  // For connecting to server running elsewhere update the base API URL
+  baseApiUrl: JSON.parse(localStorage.getItem('mifosXServerURL')) || 'https://dev.mifos.io',  // For connecting to server running elsewhere update the base API URL
+  allowServerSwitch: true, // Set to false to disable changing baseApiUrl on runtime.
```

Comment:
Privilege to switch servers can be withdrawn by setting `allowServerSwitch` to false before build time.